mtgjson5/providers/tcgplayer.py

```diff
@@ -276,11 +459,53 @@ def generate_mtgjson_sealed_product_objects(
 
         mtgjson_sealed_products = []
 
+        with constants.RESOURCE_PATH.joinpath("booster_box_size_overrides.json").open(
+            encoding="utf-8"
+        ) as f:
+            booster_box_size_overrides = json.load(f)
+
+        with constants.RESOURCE_PATH.joinpath("sealed_name_fixup.json").open(
+            encoding="utf-8"
+        ) as f:
+            sealed_name_fixup = json.load(f)
+
+        with constants.RESOURCE_PATH.joinpath("booster_box_size_overrides.json").open(
+            encoding="utf-8"
+        ) as f:
+            booster_box_size_overrides = json.load(f)
+
         for product in sealed_data:
             sealed_product = MtgjsonSealedProductObject()
+
             sealed_product.name = product["cleanName"]
+            for tag, fix in sealed_name_fixup.items():
+                if tag in sealed_product.name:
+                    sealed_product.name = sealed_product.name.replace(tag, fix)
+
             sealed_product.identifiers.tcgplayer_product_id = str(product["productId"])
             sealed_product.release_date = product["presaleInfo"].get("releasedOn")
+
+            sealed_product.category = self.determine_mtgjson_sealed_product_category(
+                sealed_product.name.lower()
+            )
+            sealed_product.subtype = self.determine_mtgjson_sealed_product_subtype(
+                sealed_product.name.lower(), sealed_product.category
+            )
+
+            LOGGER.debug(
+                f"{sealed_product.name}: {sealed_product.category.value}.{sealed_product.subtype.value}"
+            )
+
+            if sealed_product.category == MtgjsonSealedProductCategory.BOOSTER_BOX:
```

Comment:
Is this override really needed here?

Reply:
yes sometimes "draft" is not present so we need to fallback on what we know

Reply:
i'll add a comment

Reply:
oh that's not an override, it's just that the data we have only covers the booster box size